dissect/target/plugins/os/windows/regf/applications.py

```diff
@@ -20,13 +20,41 @@
     COMMON_APPLICATION_FIELDS,
 )
 
+KEYS = (
+    "HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall",
+    "HKEY_LOCAL_MACHINE\\SOFTWARE\\Wow6432Node\\Microsoft\\Windows\\CurrentVersion\\Uninstall",
+    "HKEY_CURRENT_USER\\SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Uninstall",
+)
+
 
 class WindowsApplicationsPlugin(Plugin):
     """Windows Applications plugin."""
 
-    def __init__(self, target: Target):
+    def __init__(
+        self,
+        target: Target,
+        datetime_parser: DateTimeParser = default_datetime_parser,
```

Comment:
I think with the current coding style in the Dissect projects, it's better to just use `default_datetime_parser` as a regular helper function, rather than in a constructor pattern like this. Maybe rename the helper function to `common_datetime_parser` so it has a more descriptive name.